Calibration/EcalAlCaRecoProducers/python/ALCARECOEcalESAlign_Output_cff.py

```diff
@@ -0,0 +1,31 @@
+import FWCore.ParameterSet.Config as cms
+
+
+OutALCARECOEcalESAlign_noDrop = cms.PSet(
+    # put this if you have a filter
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOEcalESAlign')
+    ),
+    outputCommands = cms.untracked.vstring(
+        'keep recoBeamSpot_offlineBeamSpot_*_*',
+
+        'keep *_ecalPreshowerDigis_*_*',
+        'keep *_ecalAlCaESAlignTrackReducer_*_*',
+
+        # 'keep recoTracks_ecalAlCaESAlignTrackReducer_*_*',
+        # 'keep recoTrackExtras_ecalAlCaESAlignTrackReducer_*_*',
+        # 'keep TrackingRecHitsOwned_ecalAlCaESAlignTrackReducer_*_*',
+        # 'keep SiStripClusteredmNewDetSetVector_ecalAlCaESAlignTrackReducer_*_*',
+        # 'keep SiPixelClusteredmNewDetSetVector_ecalAlCaESAlignTrackReducer_*_*',
+
+        # 'keep ESDigiCollection_ecalPreshowerDigis_*_*',
+        # 'keep ESDCCHeaderBlocksSorted_ecalPreshowerDigis_*_*',
+        # 'keep ESKCHIPBlocksSorted_ecalPreshowerDigis_*_*',
+
```

Comment:
2 lines is better then 8 :-)
I would prefer to have the alcareco transparent to the changes of  ecalPreshowerDigis and ecalAlCaESAlignTrackReducer. So if in the future the collections produced by those two modules change, the alCaReco should not be modified accordingly.


-- 
 							Shervin


On Tue, 5 Jan 2016, Marco Musich wrote:

> Date: Tue, 5 Jan 2016 18:03:32 +0100
> From: Marco Musich <notifications@github.com>
> Reply-To: cms-sw/cmssw
>     <reply+0045c2254727afe67f22108db7c89a9a398a205fa5532b7992cf0000000112a3b96
>     492a170ce02e9ab24@reply.github.com>
> To: cms-sw/cmssw <cmssw@noreply.github.com>
> Cc: shervin86 <shervin@cern.ch>
> Subject: Re: [cmssw] New ES Alignment: alcareco output modules,
>     ESAlignment put into stand… (#12687)
> 
>> +    outputCommands = cms.untracked.vstring(
>> +        'keep recoBeamSpot_offlineBeamSpot_*_*',
>> +
>> +        'keep *_ecalPreshowerDigis_*_*',
>> +        'keep *_ecalAlCaESAlignTrackReducer_*_*',
>> +
>> +        # 'keep recoTracks_ecalAlCaESAlignTrackReducer_*_*',
>> +        # 'keep recoTrackExtras_ecalAlCaESAlignTrackReducer_*_*',
>> +        # 'keep TrackingRecHitsOwned_ecalAlCaESAlignTrackReducer_*_*',
>> +        # 'keep SiStripClusteredmNewDetSetVector_ecalAlCaESAlignTrackReducer_*_*',
>> +        # 'keep SiPixelClusteredmNewDetSetVector_ecalAlCaESAlignTrackReducer_*_*',
>> +
>> +        # 'keep ESDigiCollection_ecalPreshowerDigis_*_*',
>> +        # 'keep ESDCCHeaderBlocksSorted_ecalPreshowerDigis_*_*',
>> +        # 'keep ESKCHIPBlocksSorted_ecalPreshowerDigis_*_*',
>> +
>
> ```$ edmEventSize -v -a EcalESAlign.root ``
>
> ``File EcalESAlign.root Events 14
> Branch Name | Average Uncompressed Size (Bytes/Event) | Average Compressed Size (Bytes/Event)
> ESDigiCollection_ecalPreshowerDigis__RECO. 14078.4 8043.86
> recoTrackExtras_ecalAlCaESAlignTrackReducer__RECO. 5486.36 2955.36
> TrackingRecHitsOwned_ecalAlCaESAlignTrackReducer__RECO. 17974.6 2436.71
> SiStripClusteredmNewDetSetVector_ecalAlCaESAlignTrackReducer__RECO. 5047.29 2432.5
> recoTracks_ecalAlCaESAlignTrackReducer__RECO. 4895.93 2066.14
> ESKCHIPBlocksSorted_ecalPreshowerDigis__RECO. 21759.6 1403.36
> SiPixelClusteredmNewDetSetVector_ecalAlCaESAlignTrackReducer__RECO. 2089.43 1184.86
> ESDCCHeaderBlocksSorted_ecalPreshowerDigis__RECO. 12854.7 1020.21
> EventProductProvenance 1677.14 211
> recoBeamSpot_offlineBeamSpot__RECO. 496.071 182.143
> EventAuxiliary 176.214 52.9286
> EventSelections 133.571 17.8571
> BranchListIndexes 62.2857 11.4286```
>
> any reason not to keep the commented list with explicit module name?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cms-sw/cmssw/pull/12687/files#r48868132
>